### ISSUE TITLE
Algunas presentaciones no se muestran correctamente.

### ISSUE THREAD

**Issue by Cyb3rG0D7** [quelea-projection/Quelea#121]
![imagen](https://user-images.githubusercontent.com/35184132/56741257-1df5fa80-6738-11e9-87e7-5dc16593d48d.png)
No sale en pantalla completa.. no hay como hacer que trabaje con powerpoint o wpsoffice o libreoffice en segundo plano y se pueda administrar la presentación con quelea mismo ?

**Comment by Cyb3rG0D7** [comment]
[TEMA 5.pptx](https://github.com/quelea-projection/Quelea/files/3198903/TEMA.5.pptx)
Les dejo el archivo del problema.

**Comment by ArvidNy** [comment]
@berry120, I just took a quick look and I actually think it might be a scaling issue in Quelea. Using the presentation above, the scaling was set to 2.0, but removing the scaling with [this line](https://github.com/quelea-projection/Quelea/blob/f9c1246d6c232008da520133c4d8fea888d76d3e/Quelea/src/main/java/org/quelea/data/powerpoint/PresentationSlide.java#L105) made the slides render correctly. I suppose those lines were made to support slides that are wider that 1920 pixels, but my guess is that it gives the opposite effect when the slide is smaller than the default, though I haven't looked at the code thoroughly enough to see exactly what it does. I believe you wrote the class, would you mind having a look at it?

**Comment by berry120** [comment]
Yup, I think you're right here - this doesn't look like the normal case of
poi not rendering correctly. I'll take a look.

**Comment by berry120** [comment]
Clearing for now as don't want to delay 2020.0 any more, and this isn't a regression.
